CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing
+
+Contributions to slide are warmly welcomed. Here's an overview of the code base of a slide and how to contribute!
+
+## Building
+Slide is super easy to build. Clone the repo and run cargo test to make sure your framework is setup. 
+Slide currently runs on the latest stable version of rust. As slide is in early development versions, it is currently
+not setup with a ui for applying rules without touching the codebase. 
```

Comment:
I think it would be helpful to provide a code example here instead of words.

```
git clone ...
cargo build
# do your development
ladder test
```

Can you elaborate what you mean by "it is currently not setup with a ui for applying rules without touching the codebase."?